src/honeysql/format.cljc

```diff
@@ -279,34 +297,52 @@
 
 (declare -format-clause)
 
+(defn map->sql [m]
+  (let [clause-ops (sort-clauses (keys m))
+        sql-str (binding [*subquery?* true
+                          *fn-context?* false]
+                  (space-join
+                   (map (comp #(-format-clause % m) #(find m %))
+                        clause-ops)))]
+    (if *subquery?*
+      (paren-wrap sql-str)
+      sql-str)))
+
+(defn seq->sql [x]
+  (if *fn-context?*
+    ;; list argument in fn call
+    (paren-wrap (comma-join (map to-sql x)))
+    ;; alias
+    (str (to-sql (first x))
+         ; Omit AS in FROM, JOIN, etc. - Oracle doesn't allow it
+         (if (= :select *clause*)
+           " AS "
+           " ")
+         (if (string? (second x))
+           (quote-identifier (second x))
+           (to-sql (second x))))))
+
 (extend-protocol ToSql
-  clojure.lang.Keyword
+  #?(:clj clojure.lang.Keyword
+     :cljs cljs.core/Keyword)
   (to-sql [x]
     (let [s (name x)]
       (case (.charAt s 0)
         \% (let [call-args (string/split (subs s 1) #"\." 2)]
              (to-sql (apply call (map keyword call-args))))
         \? (to-sql (param (keyword (subs s 1))))
         (quote-identifier x))))
-  clojure.lang.Symbol
+  #?(:clj clojure.lang.Symbol
+     :cljs cljs.core/Symbol)
   (to-sql [x] (quote-identifier x))
-  java.lang.Boolean
+  #?(:clj java.lang.Number :cljs number)
```

Comment:
Don't think this should be in here; looks like a regression from https://github.com/jkk/honeysql/pull/122